src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -479,7 +479,12 @@ protected function initializeBundles()
      */
     protected function getContainerClass()
     {
-        return $this->name.ucfirst($this->environment).($this->debug ? 'Debug' : '').'ProjectContainer';
+        $name = $this->name.ucfirst($this->environment).($this->debug ? 'Debug' : '').'ProjectContainer';
+        if (!preg_match('/^[a-zA-Z_\x7f-\xff]/', $name)) { // class name must start with a letter or underscore
```

Comment:
Instead, what about always adding a static prefix (like `Symfony`)? That would avoid executing this regex.